### ISSUE TITLE
Re-enable intergration tests on Travis

### ISSUE THREAD

**Issue by wilsonpage** [gaia-components/gaia-button#21]
@yzen I had to disable tests due to an Marionette a11y failure I didn't understand how to debug. Would you mind taking a look and re-enabling :)
